securejarhandler/src/main/java/cpw/mods/cl/ModuleClassLoader.java

```diff
@@ -201,10 +201,9 @@ protected byte[] getClassBytes(final ModuleReader reader, final ModuleReference
     private Class<?> readerToClass(final ModuleReader reader, final ModuleReference ref, final String name) {
         var bytes = maybeTransformClassBytes(getClassBytes(reader, ref, name), name, null);
         if (bytes.length == 0) return null;
-        var cname = name.replace('.', '/') + ".class";
         var modroot = this.resolvedRoots.get(ref.descriptor().name());
         ProtectionDomainHelper.tryDefinePackage(this, name, modroot.jar().getManifest(), t -> modroot.jar().getManifest().getAttributes(t), this::definePackage); // Packages are dirctories, and can't be signed, so use raw attributes instead of signed.
-        var cs = ProtectionDomainHelper.createCodeSource(toURL(ref.location()), modroot.jar().verifyAndGetSigners(cname, bytes));
+        var cs = ProtectionDomainHelper.createCodeSource(toURL(ref.location()), null);
```

Comment:
Why can't we specify the code signers anymore? iirc this is required to prevent anyone packing classes into your signed packages. Perhaps just the url is enough though?